### ISSUE TITLE
Anteprima OK, Ottieni certificato NO

### ISSUE THREAD

**Issue by anunasica** [italia/anpr#2998]
Salve, allego foto dove il problema appare chiaro:
https://postimg.cc/mcpwZyCp
le spunte funzionano, ma solo fino alla scelta della lingua (esclusa)!
Vedete l'anteprima perfettamente ottenuta, ma non funziona la spunta sulla modalità di ricezione (nel mio caso sono attive entrambe), quindi "Ottieni certificato" non si attiva.
Non so se il problema è originato dal fatto che il mio comune è entrato appena il 25 nov e quindi non ha testato a sufficienza? Grazie per un consiglio su come far funzionare le spunte. Al momento sono bloccato.
Un saluto.

**Comment by mttfranci** [comment]
Buongiorno, ha provato con un solo dispositivo o più dispositivi?
In caso può indicarci quale/quali combinazioni Dispositivo / Sistema Operativo / Browser?
Inoltre potrebbe essere utile conoscere il suo "codice identificativo anpr" riportato in visura (non è un dato particolarmente sensibile ai fini della privacy), grazie in anticipo.


![codice_identificativo_anpr](https://user-images.githubusercontent.com/29305782/144804622-f460809c-7eed-454d-ae05-b7f45472595a.png)

Saluti

Assistenza ANPR

**Comment by anunasica** [comment]
Grazie per la risposta. 
Da ieri (e solo da ieri) sul laptop w10 + firefox94 funziona perfettamente, ottima sorpresa! invece sul vetusto desktop Vista32 + firefox59ESR permane il difetto.
(magari il codice glielo invio per email?).
Su entrambi i sistemi la verifica QR è perfetta.
Grazie. Un saluto.

**Comment by mttfranci** [comment]
Salve, direi che a questo punto il suo codice non è necessario.
Cerchiamo di verificare come mai sulla combinazione windows/firefox indicata si presenta l'anomalia.
Grazie della segnalazione e buon proseguimento.
Saluti

**Comment by anunasica** [comment]
Grazie a Lei per la risposta e la disponibilità all'aiuto che ho molto apprezzato. Sì il fatto è curioso: mi aspetterei o un malfunzionamento generale (nessuna spunta disponibile) o tutto funzionante; il fatto che nei primi due quadri le spunte funzionino mentre negli ultimi due quadri no è spiazzante. Ammetto che su quella combinazione indicata priama ho difficoltà estrema con URL BLOB, ma non so se esse siano usate anche in questo Vostro sito, sicuramente lo sono nel sito ufficiale per scaricare greenpass.  Se ci sono novità ci aggiorniamo. Un saluto.

**Comment by mttfranci** [comment]
Certo, se abbiamo novità ci aggiorniamo, grazie mille per la segnalazione intanto

**Comment by anunasica** [comment]
salve, qualche novità? grazie.
